qiskit/qasm3/__init__.py

```diff
@@ -172,13 +172,60 @@
     \"\"\"
     circuit = qiskit.qasm3.loads(program)
     circuit.draw("mpl")
+
+
+Experimental import interface
+-----------------------------
+
+The import functions given above rely on the ANTLR-based reference parser from the OpenQASM project
+itself, which is more intended as a language reference than a performant parser.  You need to have
+the extension ``qiskit-qasm3-import`` installed to use it.
+
+Qiskit is developing a native parser, written in Rust, which is available as part of the core Qiskit
+package.  This parser is still in its early experimental stages, so is missing features and its
+interface is changing and expanding, but it is typically orders of magnitude more performant for the
+subset of OpenQASM 3 it currently supports.
+
+You can use the experimental interface immediately, with similar functions to the main interface
+above:
```

Comment:
Do you want to mention something about the interface not being considered stable here, and it might change (or be removed) in a future release without any warnings. The experimental api thing is new so it'd be good to highlight this in as many places as we can.

Reply:
Done in 6e729bfdd7.